MapboxDirections/MBRouteOptions.swift

```diff
@@ -8,6 +8,11 @@
 public let MBDirectionsProfileIdentifierAutomobile = "mapbox/driving"
 
 /**
+ The returned directions are appropriate for driving and incorporates current traffic conditions when determining the fastest route.
```

Comment:
Point out that this profile is experimental, like we do in MapboxGeocoder.swift for the [`locale`](https://github.com/mapbox/MapboxGeocoder.swift/blob/042bf7dcad23b84c9309f4b034b8ce5c905e59f9/MapboxGeocoder/MBGeocodeOptions.swift#L58) option.